linera-client/src/client_context.rs

```diff
@@ -464,6 +465,28 @@ where
         info!("{}", "Data blob published successfully!");
         Ok(BlobId::new_data(&blob_content))
     }
+
+    // TODO(#2490): Consider removing or renaming this.
+    pub async fn read_data_blob(
```

Comment:
Why not rename `read_data_blob` as `verify_data_blob`?

Reply:
Good question. We could do this right after onto the main branch. But I need to cherry-pick this PR on the devnet branch and renaming would break rhe rpc hash